tools/appliance/systemvmtemplate/template.json

```diff
@@ -27,8 +27,8 @@
       "format": "qcow2",
       "headless": true,
       "http_directory": "http",
-      "iso_checksum": "sha512:da7e7867ed043b784f5ae7e4adaaf4f023b5235f0fa2ead1279dc93f74bc17801ed906d330e3cd68ee8d3e96b697d21d23cfe2b755f5a9eb555bd5390a8c4dac",
-      "iso_url": "https://cdimage.debian.org/mirror/cdimage/archive/11.8.0/amd64/iso-cd/debian-11.8.0-amd64-netinst.iso",
+      "iso_checksum": "sha512:0262488ce2cec6d95a6c9002cfba8b81ac0d1c29fe7993aa5af30f81cecad3eb66558b9d8689a86b57bf12b8cbeab1e11d128a53356b288d48e339bb003dace5",
+      "iso_url": "https://cdimage.debian.org/mirror/cdimage/release/12.4.0/amd64/iso-cd/debian-12.4.0-amd64-netinst.iso",
```

Comment:
Would it maybe be better to host this on download.cloudstack.org just to be sure the URL is fully under our control?

Reply:
@wido 
make sense.
if nobody objects, I will upload to download.cloudstack.org

Reply:
ISO/packages have been uploaded to cloudstack.apt-get.eu
need some time for cdn77 to sync to download.cloudstack.org
cc @wido

Reply:
I purged the cache, URL is now available: https://download.cloudstack.org/systemvm/debian/debian-12.4.0-amd64-netinst.iso

Reply:
yeah, it works now. thanks @wido